frontend/src/App.jsx

```diff
@@ -1,11 +1,11 @@
-import Home from "./pages/Home";
-
 import "./App.css";
+import FilActu from "./components/FilActu";
 
 function App() {
   return (
     <div className="App">
-      <Home />
+
+      <FilActu />
```

Comment:
in english please, and without shorthand

Reply:
Done, replace with "NewsFeed"